tests/baseline/cases/errata-containers.yml

```diff
@@ -1779,6 +1779,7 @@ items:
 - ErratumPushItem:
```

Comment:
Could you please ensure there is at least one test which results in a non-empty container_list?

Reply:
added